modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -52,21 +55,8 @@ The *Serverless Deployment* option is displayed in the *Import from git* form on
 
 . Optional: You can use the following advanced options to further customize your application:
 
-Routing::
-Click the *Routing* link to:
-* Customize the hostname for the route.
-* Specify the path the router watches.
-* Select the target port for the traffic from the drop-down list.
-* Secure your route by selecting the *Secure Route* check box. Select the required TLS termination type and set a policy for insecure traffic from the respective drop-down lists.
-+
-For serverless applications, the Knative service manages all the routing options above. However, you can customize the target port for traffic, if required. If the target port is not specified, the default port of `8080` is used.
-
-Domain mapping::
-If you are creating a *Serverless Deployment*, you can add a custom domain mapping to the Knative service during creation.
-+
-* In the *Advanced options* section, click *Show advanced Routing options*.
-** If the domain mapping CR that you want to map to the service already exists, you can select it from the *Domain mapping* drop-down menu.
-** If you want to create a new domain mapping CR, type the domain name into the box, and select the *Create* option. For example, if you type in `example.com`, the *Create* option is *Create "example.com"*.
+include::snippets/routing-odc.adoc[]
```

Comment:
Just wondering, is there a reason that these were added as snippets? Usually I would think you'd want a snippet for things that are going to be reused a bunch of times. Plus the other sections (like "Heath checks") is not.

Reply:
Answered in Slack